bayesian/utils.py

```diff
@@ -370,6 +370,84 @@ def get_analyses_from_graph(ecosystem, package, version):
     return resp
 
 
+class GetCveByDateEcosystem:
```

Comment:
@cnulenka please change the class name based on https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html